### PR TITLE
Updated column to glimmer. Updated yeti-table to remove most use of get/set in prep for glimmer

### PR DESCRIPTION
**Pull request by cah-briangantzler**
Updated for glimmer. https://github.com/miguelcobain/ember-yeti-table/issues/344

### CODE REVIEW

File: addon/components/yeti-table/thead/row/column.js [hunk 1]
```diff
@@ -56,50 +60,98 @@ class Column extends DidChangeAttrsComponent {
...
+   * @type {Boolean}
    */
-  visible = true;
+  @dependentKeyCompat
+  get visible() {
+    return this._visible === undefined ? this.getArgWithDefault('visible', true) : this._visible;
+  }
+  set visible(value) {
+    this._visible = value;
+  }
```
Comment (miguelcobain) [inline addon/components/yeti-table/thead/row/column.js:87]: @cah-briangantzler are you aware about the `@localCopy` decorator of the https://github.com/pzuraq/tracked-toolbox project?

I think this is a perfect fit for these situations and get a rid of some boilerplate.

**Comment by cah-briangantzler** [comment]
Looks like @localCopy is not compatible with 3.16, next LTS that works is 3.20. Tracked-toolbox says it is compatible with 3.8, but its ember try scenarios do not actually test that

Stick with my boiler plate code that I will improve some, or go with this and raise min level to ember 3.20?

File: package.json [hunk 2]
```diff
@@ -40,7 +40,8 @@
     "ember-auto-import": "^1.10.1",
     "ember-cli-babel": "^7.23.1",
     "ember-cli-htmlbars": "^5.3.1",
-    "ember-decorators": "^6.1.1"
+    "ember-decorators": "^6.1.1",
```
Comment (miguelcobain) [inline package.json:43]: We can probably drop `ember-decorators` at this point, right?

**Comment by miguelcobain** [comment]
@cah-briangantzler I wouldn't mind supporting 3.20+ **or** whatever tracked-toolbox supports (as I understand it, it would be 3.8+ except 3.16).

Ember 3.20 is 9 months old at this point.

File: package.json (hunk 2 above)
Comment (cah-briangantzler) [reply package.json:43]: I think we have to wait til I convert yeti-table proper to drop ember-decorators

**Comment by cah-briangantzler** [comment]
Although tracked toolbox says 3.8+ I dont believe that is correct because of 3.16 Unless there is a fix put out, 3.20+ would be the new  min.

If you are fine with 3.20 Ill change the ember-try

**Comment by cah-briangantzler** [comment]
Weird that it looked like to did an ember try on 3.16 when I removed it from the scenarios.  

Going to close this PR and open a new one to see what happens

**Pull request closed without merging**